extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -373,6 +373,9 @@ private byte[] generateIndexHtml(String openApiPath, String swaggerUiPath, Swagg
             String oauthScopes = swaggerUiConfig.oauthScopes.get();
             options.put(Option.oauthScopes, oauthScopes);
         }
+        if (swaggerUiConfig.queryConfigEnabled) {
+            options.put(Option.queryConfigEnabled, "true");
```

Comment:
```suggestion
            options.put(Option.queryConfigEnabled, String.valueOf(swaggerUiConfig.queryConfigEnabled));
```

Reply:
Actually false is the default. It would be preferable to not set it to avoid writing the default value in the resulting HTML

Reply:
Yeah, I hesitated about this. If you're more in this camp, let's go this way.

Reply:
Ah but you're not really suggesting removing the `if` condition, then nvm. I think the current approach is good

Reply:
Oops, I misread this. I was thinking of the previous `Optional.isPresent` conditions. Ignore my suggestion :)

Reply:
Enforcing the value can also have its benefit as we are sure we push the right value if at some point they change the default. So I think I will just remove the if and enforce the value.

Reply:
@gsmet better keep it as is, we don't want to write `false` there